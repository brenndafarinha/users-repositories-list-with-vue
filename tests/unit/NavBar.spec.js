import { shallowMount } from '@vue/test-utils';
import NavBar from '../../src/components/NavBar';
import store from '../../src/store'

describe('NavBar', () => {

    it("renders correctly", () => {
        const wrapper = shallowMount(NavBar, { store });
        expect(wrapper).toMatchSnapshot();
    });

});