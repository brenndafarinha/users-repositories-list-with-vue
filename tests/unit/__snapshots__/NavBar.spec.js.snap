// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<v-app-bar dark="" max-height="80px" class="nav d-flex justify-center pa-4">
  <v-text-field solo="" clearable="" label="Buscar..." append-icon="mdi-magnify" class="mb-2"></v-text-field>
</v-app-bar>
`;
